extensions/NV/EGL_NV_context_priority_realtime.txt

```diff
@@ -0,0 +1,194 @@
+    preempting the current executing context when commands are flushed
+    by such a realtime context.
+
+    This attribute is supported only for OpenGL and OpenGL ES contexts."
+
+    Within section 3.7.4 "Context Queries" amend the eglQueryContext
+    discussion as follows:
+
+    Change the sentence describing the attribute parameter to include
+    EGL_CONTEXT_PRIORITY_LEVEL_IMG so it reads:
+
+    "attribute must be set to EGL_CONFIG_ID, EGL_CONTEXT_CLIENT_TYPE,
+    EGL_CONTEXT_CLIENT_VERSION, EGL_RENDER_BUFFER, or
+     EGL_CONTEXT_PRIORITY_LEVEL_IMG."
+
+    After the discussion of "Querying EGL_RENDER_BUFFER", add:
+
+    "Querying EGL_CONTEXT_PRIORITY_LEVEL_IMG returns the priority this
+    context was actually created with. Note: this may not be the same
+    as specified at context creation time, due to implementation limits
+    on the number of contexts that can be created at a specific priority
+    level in the system."
+
+Issues
+
+    1)  The EGL_IMG_context_priority extension is written to amend the EGL
+        1.4 specification.  Should this extension amend EGL 1.5 or 1.4?
+
+        RESOLVED:  EGL 1.5 because it is newer and better organized to
+        extend context attributes.
+
+        EGL 1.5 rewrites 3.7.1 "Creating Rendering Contexts" to have subsections
+        for different context attributes.  This extension adds a new such section
+        that includes the EGL_IMG_context_priority attribute and values too.
+
+    2)  Is context priority hint supported for both OpenGL and OpenGL ES contexts?
+
+        RESOLVED:  Yes.
+
+    3)  What is the intended application of the realtime priority level?
+
+        RESOLVED:  One anticipated application is the system compositor
+        for a Head Mounted Display (HMD) requires realtime recomposition
+        for time-warping.
+
+    4)  What action causes a context with realtime priority to preempt
+        other contexts?
+
+        RESOLVED:  Preemption by a context with realtime priority should
+        occur when there are pending rendering commands and an implicit or
+        explicit flush (i.e. glFlush or glFinish) occurs.
+
+    5)  What does "trusted" or "appropriate system privilege level"
+        mean in practice for a Linux-based operating system such as Android?
+
+        RESOLVED: Trusted means an application that has higher privileges
+        than other apps such as having CAP_SYS_NICE capability. On Android
+        such applications have to be registered in advance with the OS;
+        unpriviledged third party app cannot acquire this capability.
+
+        This restriction exists so arbitrary applications do not starve or
+        otherwise compromise the interactivity of the system overall.
+
+    6)  In practice how many realtime priority contexts can exist in a system?
+
+        RESOLVED: Only one realtime priority context should be active at one moment
+        to get best performance.
```

Comment:
This doesn't really answer the question; the lack of answer kind of implies there is no limit, but I think you should answer explicitly the question you're asking

Reply:
I think I should rephrase the question: "In practice how many realtime priority contexts can exist in a system to get best performance of it?". 

Because it's just a guideline on how many should exists for best performance.